### DISCUSSION TITLE
Image preview

### DISCUSSION THREAD

**Discussion by scomnoob** [windirstat/windirstat#139] [Ideas]
Hello

Any chance to add image view inside program ?

Its very annoying to chech thousands of folders...

**Comment by NoMoreFood** [comment]
It's possible we could do something like the Windows Preview Pane (implemented with IPreviewHandler).  What is your specific use case?  Are you just trying to clean up old files and using WinDirStat to prioritize your clean up by size?

**Reply by scomnoob** [reply]
Yes
There are some app generating images in multiple folders - i need to check them .
Biggest priority - folders with multiple files
